kotest-assertions/kotest-assertions-json/src/jvmMain/kotlin/io/kotest/assertions/json/JsonUtil.kt

```diff
@@ -0,0 +1,7 @@
+package io.kotest.assertions.json
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+
+// we need to make it public because inline functions use it
+val mapper by lazy { ObjectMapper().registerKotlinModule() }
```

Comment:
```suggestion
private val mapper by lazy { ObjectMapper().registerKotlinModule() }
@PublishedApi
internal fun getMapper() = mapper
```

I think this works. I rather not have this mapper leak to users' code

Reply:
I've changed your suggestion a bit to avoid names collision of symbols and commited as 
3ded3ce.